src/main/java/ar/edu/unq/sarmiento/wicket/materia/ListadoDeMateriasPage.html

```diff
@@ -22,11 +22,17 @@ <h4>
 						<td wicket:id="docente"></td>
 						<td wicket:id="correlativas"></td>
 						<td>
-							<button wicket:id="correlativa" class="btn btn-warning">Agregar Correlativa</button>
-							<button wicket:id="borrarCorrelativa" class="btn btn-warning">Borrar Correlativa</button>
+							<button wicket:id="correlativa" class="btn btn-warning">
+								<span class="glyphicon glyphicon-plus" title="Agregar correlativa"> correlativa</span>
+							</button>
+							<button wicket:id="borrarCorrelativa" class="btn btn-warning">
+								<span class="glyphicon glyphicon-minus"title="Borrar correlativa"> correlativa</span>
```

Comment:
Ojo, según la documentación de bootstrap no va nada dentro del  `<span>`. Sería así:
```suggestion
								<span class="glyphicon glyphicon-minus"title="Borrar correlativa"></span> correlativa
```

Reply:
Listo esto lo dejé afuera y quedó mejor, gracias.
![Captura de pantalla de 2019-06-17 22-44-01](https://user-images.githubusercontent.com/32805369/59647410-77cfdb00-9151-11e9-9f93-f393c9f18386.png)